gradle/libs.versions.toml

```diff
@@ -127,8 +127,10 @@ jcasbin = "1.81.0"
 ognl = "3.4.7"
 concurrent-trees = "2.6.0"
 jakarta-validation = "2.0.2"
+aspectj = "1.9.24"
 
 [libraries]
+aspectj-aspectjrt = { group = "org.aspectj", name = "aspectjrt", version.ref = "aspectj" }
```

Comment:
Can you please 1. check the license of this dependency; 2. Update the license.bin to add this; 3. check the transitive dependencies of this one, and exclude the conflicted/unused ones if necessary.

Reply:
Updated the license; AspectJ does not have third-party dependencies.